errors/handler.go

```diff
@@ -27,25 +27,33 @@ type StackFrames struct {
 	Package    string
 }
 
+var MarkSpanAsError = errors.New("")
+
 // Write exception event in span using the recover data from panic
 func LogError(span opentracing.Span, recoverData interface{}, skipFrames int) {
 	var exceptionFields = getExceptionLogFields(recoverData, skipFrames+1)
 	span.LogFields(exceptionFields...)
 	span.SetTag("error", true)
 }
-func LogErrorInRawSpan(rawSpan *tracer.RawSpan, recoverData interface{}) {
-	var exceptionFields = getExceptionLogFields(recoverData, 1)
-	if rawSpan.Logs == nil {
-		rawSpan.Logs = []opentracing.LogRecord{}
-	}
+
+func LogErrorInRawSpan(rawSpan *tracer.RawSpan, err **errors.Error) {
 	if rawSpan.Tags == nil {
 		rawSpan.Tags = opentracing.Tags{}
 	}
-	rawSpan.Logs = append(rawSpan.Logs, opentracing.LogRecord{
-		Timestamp: time.Now(),
-		Fields:    exceptionFields,
-	})
-	rawSpan.Tags["error"] = true
+	if *err == MarkSpanAsError {
+		rawSpan.Tags["error"] = true
+	} else {
+		var exceptionFields = getExceptionLogFields(*err, 1)
+		if rawSpan.Logs == nil {
+			rawSpan.Logs = []opentracing.LogRecord{}
+		}
+		rawSpan.Logs = append(rawSpan.Logs, opentracing.LogRecord{
+			Timestamp: time.Now(),
+			Fields:    exceptionFields,
+		})
+		rawSpan.Tags["error"] = true
+		*err = MarkSpanAsError
```

Comment:
Is this the exception that gets reported on the CLI when the tests fail?

Reply:
Go is clever enough to append all panic/recover pairs, so in this example the panic standard output is: 
![image](https://user-images.githubusercontent.com/69803/72667438-4e936800-3a1c-11ea-8a4b-940d8e29bdef.png)

Reply:
None of those frames include the one that originally panicked, right?

Reply:
It's included: 
![image](https://user-images.githubusercontent.com/69803/72667541-fad54e80-3a1c-11ea-87d0-4fd9346c4fd2.png)

Reply:
But also we have all the `Finish` calls of the parent spans, do you prefer to avoid that frames? maybe I can store the stacktrace when the panic happens.

Reply:
It's a lot of noise, but I think it's OK for now